test/windows.jl

```diff
@@ -40,6 +49,22 @@ end
     kaiser_jl = kaiser(128, 0.4/π)
     kaiser_ml = readdlm(joinpath(dirname(@__FILE__), "data", "kaiser128,0.4.txt"), '\t')
     @test kaiser_jl ≈ kaiser_ml
+
+    gaussian_jl = gaussian(128, 0.2)
+    gaussian_ref = readdlm(joinpath(dirname(@__FILE__), "data", "gaussian128,0.2.txt"), '\t')
+    @test gaussian_jl ≈ gaussian_ref
```

Comment:
Could also test against `w(x) = exp(-0.5*(x/sigma)^2); w.(linspace(-0.5, 0.5, n)` to verify it matches the docstring.